### PR TITLE
Update JanusGraph 0.4 and use testcontainers

### PR DESCRIPTION
**Pull request by farodin91**

### CODE REVIEW

**Comment by marco-scoppetta** [comment]
Hey @farodin91 ,

have you tested these changes?
I have no experience with test containers, but testing your version I get the following:

```
java.lang.UnsatisfiedLinkError: /private/var/folders/8x/4qdbl1_912sc_h2ffjd7wf7m0000gp/T/fdbjni3496103459786540371.library: dlopen(/private/var/folders/8x/4qdbl1_912sc_h2ffjd7wf7m0000gp/T/fdbjni3496103459786540371.library, 1): Library not loaded: libfdb_c.dylib
  Referenced from: /private/var/folders/8x/4qdbl1_912sc_h2ffjd7wf7m0000gp/T/fdbjni3496103459786540371.library
  Reason: image not found

	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1824)
	at java.lang.Runtime.load0(Runtime.java:809)
	at java.lang.System.load(System.java:1086)
	at com.apple.foundationdb.JNIUtil.loadLibrary(JNIUtil.java:106)
	at com.apple.foundationdb.FDB.<clinit>(FDB.java:97)
```
this happens when initialising `FoundationDBStoreManager`, any idea?

**Comment by farodin91** [comment]
You have to install the foundationdb driver before.

**Comment by marco-scoppetta** [comment]
@farodin91 the FoundationDBStoreManager already uses foundationdb driver, ive updated the POM to use:
```
<groupId>org.foundationdb</groupId>
<artifactId>fdb-java</artifactId>
```
am I missing something else?

**Comment by nvvalchev** [comment]
Hello, are there are plans for merging this PR?

**Comment by theZiki** [comment]
Hi, when we can expect that this will be merged? JanusGraph is now on 0.5.0 version, it's two major versions behind? 
Tnx

**Comment by farodin91** [comment]
@theZiki I don't think will be merged in the current state. 

Please stay tuned, @rngcntr is working on an improved version.

**Comment by rngcntr** [comment]
I can confirm what @farodin91 says ;)

**Comment by theZiki** [comment]
Can I and my team help you? We are using this in our company and we want to contribute and stay tuned :-)

**Comment by rngcntr** [comment]
I'm currently working on a [fork](https://github.com/rngcntr/janusgraph-foundationdb) based on this PR from @farodin91.
It already conatins some improvements and you are welcome to contribute

**Comment by theZiki** [comment]
Do we have a chat on Gitter?

**Comment by rngcntr** [comment]
> Do we have a chat on Gitter?

I contacted one of the JanusGraph maintainers and asked him to provide a room in their JanusGraph community on Gitter

**Comment by FlorianHockmann** [comment]
> Do we have a chat on Gitter?

JanusGraph has currently only [one chat on Gitter](https://gitter.im/janusgraph/janusgraph) but that is mainly used for usage discussions. I think that at least a chat dedicated for development discussions would be helpful and maybe a dedicated chat for this FoundationDB backend, so I [suggested that on the janusgraph-dev list](https://groups.google.com/forum/#!topic/janusgraph-dev/5Fp2tQNn_Po). Feel free to voice your opinion their. Until we have dedicated rooms for development, you can of course use the existing JanusGraph room.

**Comment by theZiki** [comment]
@farodin91 any update about new version of janus?

**Comment by rngcntr** [comment]
@theZiki We have a new [chat room on Gitter ](https://gitter.im/janusgraph/janusgraph-dev) which we can use to discuss the FoundationDB storage adapter (among others)

**Comment by theZiki** [comment]
> 
> 
> I'm currently working on a [fork](https://github.com/rngcntr/janusgraph-foundationdb) based on this PR from @farodin91.
> It already conatins some improvements and you are welcome to contribute

Hi, I want to contribute, can you please direct me what to do?

**Comment by rngcntr** [comment]
First of all, thanks for your offer!
I didn't work on the project the last few months and there are still two open issues. If you think you can help me with this, feel free to do so ;)

We are currently discussing official support of FoundationDB in JanusGraph over [here](https://groups.google.com/forum/#!topic/janusgraph-dev/LS75m3rXZss) and [here](https://groups.google.com/forum/#!topic/janusgraph-dev/tleruwXYBzU). From what I see, help is also appreciated there

File: src/test/java/com/experoinc/janusgraph/FoundationDBContainer.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
+// Copyright 2018 Expero Inc.
```
Comment (mbrukman) [inline src/test/java/com/experoinc/janusgraph/FoundationDBContainer.java:1]: Update to "JanusGraph Authors" since this is now part of JanusGraph org?

**Pull request closed without merging**